### ISSUE TITLE
Widget areas not ready for internationalization

### ISSUE THREAD

**Issue by mybooking-es** [mybooking-es/mybooking-wp-theme#13]
Widget areas are not ready for internationalization. 

see inc/widgets.php

``
	register_sidebar(
		array(
			'name'          => __( 'Menú de navegación', 'mybooking' ),
			'id'            => 'mybooking_primary_menu',
	    'description'    => __( 'Área de widgets a continuación del menú principal', 'mybooking' ),
			'before_widget' => '<div>',
			'after_widget'  => '</div>',
		)
	);
``

**Comment by cranca** [comment]
They are, in fact they are translated right now.

![image](https://user-images.githubusercontent.com/1257187/79224654-c75b6200-7e5b-11ea-9c5d-aec4a3a17f6b.png)

But it's true that some of translations seems to be lost, despite I remenber have done it... I have to rework it.

I used this i18n scheme because in some areas like options pages and custom post types any other  scheme (I tried _e() and _ex() with no luck) makes the labels desapear.
